docs/CONFIGURATION.md

```diff
@@ -0,0 +1,56 @@
+This document provides details on how to configure the application, manage environment variables, and use the VEDA UI configuration provider.
+
+## Environment variables
+
+The application uses `.env` and `.env.local` files to manage environment variables. Below is a breakdown of the variables used in the application:
```

Comment:
can we encourage users to copy `.env.local-copy` since this would have essential variables to run the application? + a short sentence about the differentiation between `.env` and `.env.local` (information that should note be exposed to public) might be helpful.